src/AppLovinMAXAdView.js

```diff
@@ -21,6 +21,34 @@ export const AdViewPosition = {
   BOTTOM_RIGHT: "bottom_right",
 };
 
+const AdViewWrapper = (props) => {
+  const {style, ...rest} = props;
+  return (
+    AppLovinMAX.isInitialized() ?
+      <AdView {...style} {...rest}/>
+    :
+      <View style={[styles.container, style]} {...rest}>
+        {
+          console.warn('[AppLovinSdk] [AppLovinMAX] <AdView/> has been mounted before AppLovin initialization')
+        } 
+        <Text style={styles.message}>Ad placeholder by AppLovin &copy;</Text>
+      </View>
+  )
+};
+
+const styles = StyleSheet.create({
+  container: {
+    flex:1,
```

Comment:
```suggestion
    flex: 1,
```

Reply:
Just updated the PR messages.   It is not easy to print a message via AdViewManager so I just used console.warn() instead.   Also, I'm not sure we should put any message on the black screen.